processor/processor.go

```diff
@@ -487,15 +539,18 @@ func (proc *HandleT) processJobsForDest(jobList []*jobsdb.JobT, parsedEventList
 
 	//Now do the actual transformation. We call it in batches, once
 	//for each destination ID
+	//logger.Debug("=== calling transformations ===")
 	for destID, destEventList := range eventsByDest {
 		//Call transform for this destination. Returns
 		//the JSON we can send to the destination
 		url := integrations.GetDestinationURL(destID)
+		//logger.Debug("=== url ===", url)
 		logger.Debug("Transform input size", len(destEventList))
 		response := proc.transformer.Transform(destEventList, url, transformBatchSize)
 		destTransformEventList := response.Events
 		logger.Debug("Transform output size", len(destTransformEventList))
 		if !response.Success {
+			logger.Error("=== response not ok ===")
```

Comment:
Let's print the error with logger.Debug and add more error info.

Reply:
done